app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    if isinstance(command, str) and command.split(" ")[0] == "cp":
+        file_name = command.split(" ")[1]
```

Comment:
The variable `file_name` could be more descriptive. Consider renaming it to `source_file` for better readability and consistency with `destination_file`.